app/src/main/java/org/mozilla/fenix/experiments/NimbusSetup.kt

```diff
@@ -66,6 +67,10 @@ fun createNimbus(context: Context, url: String?): NimbusApi =
                 globalUserParticipation = enabled
             }
 
+            if (url.isNullOrBlank()) {
```

Comment:
Ah nice! Should the entry in `nimbus.remote-settings.url=` still work as well? I am not getting any experiments back locally anymore in a debug build.

Reply:
Yes, this still works. @jhugman clarified for me.